index.html

```diff
@@ -9,6 +9,12 @@
     <link rel="stylesheet" type="text/css" href="scss/application.css">
 </head>
 <body>
-    <h1 class="title">Hello World</h1>
+    <section>
+        <section>
+          <header class="header-title">
+            <h1 class="title">Front-End Tools</h1>
```

Comment:
Lets leave this tag alone and use a pseudo-element in css to add the image and reduce html code